### ISSUE TITLE
Stacking Flagallation buff gets removed immediately.

### ISSUE THREAD

**Issue by fuu1** [SimCMinMax/WoW-BugTracker#899]
**Game Version:**
> 9.2

**Description:**
> The stacking buff applied from Flagellation gets very rarely removed instantly disallowing you to continue stacking or generate benefits based on stacks. The assumption is that this is caused by a issue in a server sided script.

**Logs, Images, Videos**
https://www.warcraftlogs.com/reports/GwTQfHpznarDF62y/#fight=last&phase=1&type=auras&source=9
https://www.warcraftlogs.com/reports/zCy6K7vVh2NXMTAZ#fight=6&type=auras&source=13&view=timeline
https://www.warcraftlogs.com/reports/p3hRrQcXnZ4gfFBT#fight=13&type=auras&source=7&view=events&ability=323654

**Reproducible:**
>  hard to reproduce

**Comment by RosvallM** [comment]
Remade this comment - 

I had some success in triggering this bug by activating the Axe mode from Cache of Acquired Treasures

/cast Flagellation(Venthyr)
/use Cache of Acquired Treasures  

However I would not call this reproducible. While I initially thought that, since I got the bug twice in a row from never having had it before. When I tried to log it, I haven't been able to get it to happen again over 7 additional tries.

Setup was in Venthyr class hall on a training dummy, without any additional setup.
